plugins/inputs/win_services/win_services_test.go

```diff
@@ -201,6 +201,7 @@ var testSimpleData = []testData{
 	{[]string{"Service 1", "Service 2"}, nil, nil, []serviceTestInfo{
 		{nil, nil, nil, "Service 1", "Fake service 1", 1, 2},
 		{nil, nil, nil, "Service 2", "Fake service 2", 1, 2},
+		//{nil, nil, nil, "NoService 3", "Fake service 3", 1, 2},
```

Comment:
Is this some left-over? If not, it could need a comment on why it is there but commented. :-)

Reply:
Good catch, not sure what the purpose of it was. removed it.